x-pack/metricbeat/module/googlecloud/compute/_meta/data_disk.json

```diff
@@ -5,15 +5,16 @@
             "id": "elastic-observability"
         },
         "instance": {
-            "id": "7692260917184259934",
-            "name": "instance-test-ks"
+            "id": "7925831082803123728",
+            "name": "heartbeat-tokyo"
         },
         "machine": {
-            "type": "e2-standard-2"
+            "type": "f1-micro"
         },
         "provider": "googlecloud"
     },
-    "cloud.availability_zone": "us-central1-a",
+    "cloud.availability_zone": "asia-northeast1-b",
+    "cloud.region": "asia-northeast1",
```

Comment:
Nothing to change here, just thinking loud. It might be interesting to have common values for regions. So a user can more easily group metrics coming from `asia-northeast1` in GCP, and `ap-northeast-1` in AWS.

Imagine for example being able to ingest with something like this metadata:
```
"cloud.region": {
    "name": "asia-northeast1",
    "area": "asia-pacific",
    "country": "japan",
    "city": "tokyo",
}
```
And if it were ingested from AWS:
```
"cloud.region": {
    "name": "ap-northeast-1",
    "area": "asia-pacific",
    "country": "japan",
    "city": "tokyo",
}
```

Though maybe for that we can [add geodata](https://www.elastic.co/guide/en/elasticsearch/reference/7.9/geoip-processor.html) through an ingest pipeline in the future.

Reply:
@jsoriano Yeah good point! Different cloud has different names for regions. I don't see a specific API for GCP though but I think the geoip processor should be sufficient if we see good use cases in the future!